codalab/lib/bundle_cli.py

```diff
@@ -662,12 +662,34 @@ def print_table(
         """
         Pretty-print a list of columns from each row in the given list of dicts.
         """
-        # Get the contents of the table
         rows = [columns]
+        # display restricted fields if the server returns those fields - which suggests the user is root
```

Comment:
We should add a test for running this command with a root user and a non-root user to ensure that restricted fields are displayed only to root.

Also, can you document this feature (different formats for root and non-root users) in the CLI command documentation?